### PR TITLE
Jcs/rate object code

### PR DESCRIPTION
**Pull request by jSylvestre**
I don't know that we want to go this way.
Another way we could do it is change the billing unit to a dropdown then base the object code off of RR, PLS, or CAES. And PassThrough will have separate values

If we do this, we should reset the DB after deployment. Also, manually set the see value to invoices to 1000 in test to avoid future sloth duplicates

### CODE REVIEW

File: Harvest.Core/Domain/Expense.cs [hunk 1]
```diff
@@ -59,6 +59,9 @@ public class Expense
         [StringLength(50)]
         public string Account { get; set; }
 
+        //This is actually required, but that will make the migration a pain...
+        public string ExpenseObjectCode { get; set; }
```
Comment (sprucely) [inline Harvest.Core/Domain/Expense.cs]: After generating the migration, it is okay to manually edit the corresponding code to first create the column as nullable, update with a default value, then change it to not nullable. But yeah, it is annoying.

**Review by sprucely — APPROVED** [review]

File: Harvest.Core/Domain/Expense.cs (hunk 1 above)
Comment (sprucely) [reply Harvest.Core/Domain/Expense.cs]: oops. Just noticed it's just a draft.

Comment (jSylvestre) [reply Harvest.Core/Domain/Expense.cs]: The update would be a pain. This will need to wipe the DB for proper values first.
And I'll just make sure this is set with code.

**Comment by srkirkland** [comment]
@jSylvestre I haven't actually read through this PR yet but from first sight we shouldn't be adding anything specifically for ObjectCode.  That ties us too much to KFS which is going away soon.  I don't know if AE will even have them.  I think it's better to fit all of the account info into the account field.  I have some docs I need to read through about account formats that include obj.

**Comment by jSylvestre** [comment]
Which is why this is better :)
It isn't tied to the account, it is a separate field we can ignore in the future, also, it isn't for this account, but the account it comes from.

**Comment by jSylvestre** [comment]
This will not be used, but keeping the code around as an example
